slackbot/slackclient.py

```diff
@@ -35,16 +37,15 @@ def __init__(self, token, timeout=None, bot_icon=None, bot_emoji=None, connect=T
         self.rtm_start_args = rtm_start_args
 
         if timeout is None:
-            self.webapi = slacker.Slacker(self.token)
+            self.webapi = slack_sdk.WebClient(self.token)
         else:
-            self.webapi = slacker.Slacker(self.token, timeout=timeout)
+            self.webapi = slack_sdk.WebClient(self.token, timeout=timeout)
```

Comment:
```suggestion
            self.webapi = slack_sdk.WebClient(self.token, timeout=timeout)
        # This handler does retries when HTTP status 429 is returned
        from slack_sdk.http_retry.builtin_handlers import RateLimitErrorRetryHandler
        rate_limit_handler = RateLimitErrorRetryHandler(max_retry_count=100)
        # Enable rate limited error retries as well
        self.webapi.retry_handlers.append(rate_limit_handler)
```

I'd recommend we register the builtin `RateLimitErrorRetryHandler` here

Reply:
done